packages/components/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,91 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Slot as RadixSlot } from '@radix-ui/react-slot';
+import { Children, forwardRef, useState, type ForwardedRef, type ReactNode } from 'react';
+
+import { ForwardedRefColorGradientInput } from './ColorGradientInput';
+import { ForwardedRefColorPickerInput } from './ColorPickerInput';
+import { ForwardedRefColorValueInput } from './ColorValueInput';
+import styles from './styles/customColorPicker.module.scss';
+import { type ColorFormat, type RgbaColor } from './types';
+import { DEFAULT_COLOR, DEFAULT_FORMAT, getColorWithName } from './utils';
+
+type ColorPickerProps = {
+    /**
+     * The children of the color picker component. This can contain multiple `ColorPicker.Values` or `ColorPicker.Gradient` components.
+     */
+    children?: ReactNode;
+    /**
+     * The active color in the color picker
```

Comment:
maybe we can add a default here?

Reply:
not necessary, as we still need the defaults in the subcomponents. If we make the prop required in the subcomponents, typescript will complain because it doesn't know about the slot that injects these.